### PR TITLE
Add new maven and gradle repositories to examples

### PR DESCRIPTION
**Pull request by Barry-RG**

### CODE REVIEW

File: documentation/usage/api/index.md [hunk 1]
```diff
@@ -114,30 +126,70 @@ be migrated to a state the rest of the code can work with.
...
+        &lt;url&gt;https://download.red-gate.com/maven/release&lt;/url&gt;
+    &lt;/repository&gt;
+    ...
+&lt;/repositories&gt;
+&lt;dependencies&gt;
+    ...
+    &lt;dependency&gt;
+        &lt;groupId&gt;org.flywaydb<strong>.enterprise</strong>&lt;/groupId&gt;
+        &lt;artifactId&gt;flyway-core&lt;/artifactId&gt;
+        &lt;version&gt;{{ site.flywayVersion }}&lt;/version&gt;
```
Comment (DoodleBobBuffPants) [inline documentation/usage/api/index.md]: I think this should be `site.flywayPreviewVersion`?

File: documentation/usage/api/index.md [hunk 2]
```diff
@@ -114,30 +126,70 @@ be migrated to a state the rest of the code can work with.
...
         <td>
-            <pre class="prettyprint">compile "org.flywaydb<strong>.enterprise</strong>:flyway-core:{{ site.flywayVersion }}"</pre>
+            <pre class="prettyprint">repositories {
+    mavenCentral()
+    maven {
+        url "https://download.red-gate.com/maven/release"
+    }
+}
+dependencies {
+compile "org.flywaydb<strong>.enterprise</strong>:flyway-core:{{ site.flywayVersion }}"
```
Comment (DoodleBobBuffPants) [inline documentation/usage/api/index.md]: I think this should be `site.flywayPreviewVersion`?

File: documentation/usage/maven/index.md [hunk 3]
```diff
@@ -30,19 +30,27 @@ The Flyway Maven plugin supports **Maven 3.x** running on **Java 8**, **Java 9**
...
-    &lt;groupId&gt;org.flywaydb&lt;/groupId&gt;
-    &lt;artifactId&gt;flyway-maven-plugin&lt;/artifactId&gt;
-    &lt;version&gt;{{ site.flywayPreviewVersion }}&lt;/version&gt;
-&lt;/plugin&gt;</pre>
+            <pre class="prettyprint">&lt;build&gt;
+    ...
+    &lt;plugin&gt;
+        &lt;groupId&gt;org.flywaydb&lt;/groupId&gt;
+        &lt;artifactId&gt;flyway-maven-plugin&lt;/artifactId&gt;
+        &lt;version&gt;{{ site.flywayVersion }}&lt;/version&gt;
```
Comment (DoodleBobBuffPants) [inline documentation/usage/maven/index.md:31]: I think this should be `site.flywayPreviewVersion`?

File: documentation/usage/maven/index.md [hunk 4]
```diff
@@ -59,19 +67,43 @@ The Flyway Maven plugin supports **Maven 3.x** running on **Java 8**, **Java 9**
...
+        &lt;url&gt;https://download.red-gate.com/maven/release&lt;/url&gt;
+    &lt;/repository&gt;
+    ...
+&lt;/repositories&gt;
+&lt;build&gt;
+    ...
+    &lt;plugin&gt;
+        &lt;groupId&gt;org.flywaydb<strong>.enterprise</strong>&lt;/groupId&gt;
+        &lt;artifactId&gt;flyway-maven-plugin&lt;/artifactId&gt;
+        &lt;version&gt;{{ site.flywayVersion }}&lt;/version&gt;
```
Comment (DoodleBobBuffPants) [inline documentation/usage/maven/index.md]: I think this should be `site.flywayPreviewVersion`?

**Review by DoodleBobBuffPants — CHANGES_REQUESTED** [review]

**Comment by angellaa** [comment]
We need to add legal text on each page close to the point when the user download/use the teams/enterprise APIs...

Alongs the lines:
By "downloading Flyway Teams" you confirm that you have read and agree to the terms of the [Redgate EULA](https://www.red-gate.com/assets/purchase/assets/subscription-license.pdf?_ga=2.265045707.556964523.1656332792-1685764737.1620948215).

File: documentation/usage/api/index.md [hunk 5]
```diff
@@ -92,54 +70,46 @@ be migrated to a state the rest of the code can work with.
...
-        <td>
-            <pre class="prettyprint">compile "org.flywaydb<strong>.enterprise</strong>:flyway-core:{{ site.flywayVersion }}"</pre>
+            <pre class="prettyprint">repositories {
+    mavenCentral()
+    maven {
+        url "https://download.red-gate.com/maven/release"
+    }
+}
+dependencies {
+compile "org.flywaydb<strong>.enterprise</strong>:flyway-core:{{ site.flywayVersion }}"
```
Comment (DoodleBobBuffPants) [inline documentation/usage/api/index.md]: I think this `compile` line needs to be spaced in a bit so it's indented w.r.t. `dependencies`

**Review by DoodleBobBuffPants — CHANGES_REQUESTED** [review]
All looks good, one minor thing about spacing

**Review by DoodleBobBuffPants — APPROVED** [review]

**Review by madninjaskillz — APPROVED** [review]

**Pull request merged**